crates/avalanche-types/src/subnet/rpc/snow/engine/common/vm.rs

```diff
@@ -40,6 +40,7 @@ pub trait CommonVm: AppHandler + Connector + Checkable {
         to_engine: Sender<Message>,
         fxs: &[Fx],
         app_sender: Self::AppSender,
+        warp_signer: Self::WarpSigner,
```

Comment:
You could just pass in the concrete type here. It's unclear if there would ever be any associated type other than `SignerClient<Channel>` that was ever used.